eclair-core/src/main/scala/fr/acinq/eclair/wire/ChannelCodecs.scala

```diff
@@ -266,12 +266,35 @@ object ChannelCodecs extends Logging {
       ("shortChannelId" | shortchannelid) ::
       ("lastSent" | fundingLockedCodec)).as[DATA_WAIT_FOR_FUNDING_LOCKED]
 
+  // All channel_announcement's written prior to supporting unknown trailing fields had the same size
+  val noUnknownFieldsChannelAnnouncementSizeCodec: Codec[Int] = provide(430)
```

Comment:
Comment should also explain that this works because we're only reading channel announcements that we create ourselves, with an empty `features` field

Reply:
Very good point, I missed that completely.

Reply:
Fixed with 02b23f4